Setting up Privacy Policy

Danny-You opened the pull request:
As a user
I can see a privacy policy
So that I know how my data will be used

Closes #6 

![image](https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-3/assets/91991737/cbfaea1a-e6d3-451f-8d68-82f6a141e3ce)

Danny-You committed:
dy - set up privacy policy

lawrencewang30 reviewed (dismissed):
Code and screenshot looks good to me, can merge this branch after others send code review

pconrad reviewed (dismissed):
The issue asked for the privacy policy link to be in the footer, not a new button in the header.

The location next to the Login button is kind of distracting.

Can we remove that, and instead put the link in the footer where the issue requested it?

Danny-You committed:
dy - set up privacy policy

Danny-You commented:
Addressed the issue, removed the button for Privacy Policy in the header, added the link in the footer where the issue requested it. Please give it another review.

![image](https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-3/assets/91991737/cbfaea1a-e6d3-451f-8d68-82f6a141e3ce)

Kanagawa-okiNamiUra reviewed (approved):
Checked the running instance at Danny's localhost; looks good to me

InariHimeko reviewed (approved):
All the code LGTM

GDW1 reviewed (approved):
This PR seems good to me

chriscerie commented on frontend/src/main/components/Nav/Footer.js:
```diff
@@ -44,6 +44,9 @@ export default function Footer() {
           .
         </p>
+        <p>
+          <a href="/privacy">Privacy Policy</a>
```
Use react-router’s `Link` over anchor tags to avoid full page reloads.

pconrad replied:
Agreed with Chris.

pconrad commented:
I'm going to approve this, but I'm also going to ask you to fix this in another PR.

<img width="859" alt="image" src="https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-3/assets/1119017/a049c004-92bd-4f16-a104-a9de307aacd5">
